tasks/release.py

```diff
@@ -1049,3 +1049,55 @@ def create_qa_cards(ctx, tag):
         return
     setup_ddqa(ctx)
     ctx.run(f"ddqa --auto create {version.previous_rc_version()} {tag} {get_labels(version)}")
+
+
+@task
+def create_github_release(_ctx, version, draft=True):
+    """
+    Create a GitHub release for the given tag.
+    """
+    import pandoc
+
+    sections = (
+        ("Agent", "CHANGELOG.rst"),
+        ("Datadog Cluster Agent", "CHANGELOG-DCA.rst"),
+    )
+
+    notes = []
+
+    for section, filename in sections:
+        text = pandoc.write(pandoc.read(file=filename), format="markdown_strict")
```

Comment:
When running pandoc this way it creates the output with the wrapped lines, which look bad in the github release, so we pass `--wrap=none` when running this manually. Do you know if the result is the same when called from the code and if so - can we add such flag?

Reply:
Ah true I forgot this one! Yes we can pass it. Thanks